frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -81,12 +81,7 @@ export default function SideNavigation({
 
   const doesNotContainUserApproval = useCallback(
     (review: ReviewRequestInterface) => {
-      return (
-        currentUser &&
-        !review.responses.find(
-          (response) => response.user === `user:${currentUser.dn}` && response.decision === Decision.Approve,
-        )
-      )
+      return currentUser && !review.responses.find((response) => response.user === `user:${currentUser.dn}`)
```

Comment:
We're no longer looking for approvals, is this a mistake?

Reply:
See above